filter_plugins/get_attr.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
```

Comment:
```bash
$ pycodestyle get_attr.py
get_attr.py:15:1: E302 expected 2 blank lines, found 1
get_attr.py:20:1: E302 expected 2 blank lines, found 0
```

I would suggest adding this to analysis. However, it seems like this plugin is not going to be used only in this role neither is it going to be the only one we'll need. As such, perhaps it would be a good idea to create a repo to serve as a collection of plugins. There, we could add tests for each of them. Thoughts?